src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resource/TemporaryAbsencesResourceTest.kt

```diff
@@ -120,4 +125,28 @@ class TemporaryAbsencesResourceTest : IntegrationTestBase() {
       verify(temporaryAbsenceService).getTemporaryAbsence("MDI", "A1234AA")
     }
   }
+
+  @Nested
+  inner class `Confirm arrival from temporary absence` {
+
+    @Test
+    fun `confirm arrival`() {
+      whenever(temporaryAbsenceService.temporaryAbsencesArrival(eq("G5666UK"), any())).thenReturn(InmateDetail(1L))
+      val token = getAuthorisation(roles = listOf("ROLE_VIEW_ARRIVALS"), scopes = listOf("write"))
+      val temporaryAbsencesArrival = TemporaryAbsencesArrival(
+        "NMI",
+        "ET",
+        "Comment",
+        LocalDateTime.of(2021, 11, 15, 1, 0, 0)
+      )
+
+      webTestClient
```

Comment:
we usually suggest stubbing very loosely:

```
whenever(temporaryAbsenceService.temporaryAbsencesArrival(any(), any()))`
```

 and then being very specific when verifying the service
```
verify(temporaryAbsenceService). temporaryAbsencesArrival("MDI", temporaryAbsencesArrival)
```

Reply:
fixed